anagrafica/templates/pdf_estensione.html

```diff
@@ -20,12 +20,19 @@
 <P ALIGN=RIGHT STYLE="margin-bottom: 0.35cm"><BR><BR>
 </P>
 <P ALIGN=LEFT STYLE="margin-bottom: 0.35cm">{{ estensione.creazione }}</P>
-<div align="right" style="margin-top: 10px; padding-top: 10px align: right;">
+{% if not estensione.automatica %}
+    <div align="right" style="margin-top: 10px; padding-top: 10px align: right;">
 			<span style="text-align: right; position: relative; right;">
 				In Fede <br/>
 			</span>
 			<span style="text-align: right; position: relative; right;">
 				F.to {{ estensione.persona.nome }} {{ estensione.persona.cognome }}
 			</span>
 		</div>
+{% else %}
+    <div style="margin-top: 10px; padding-top: 10px align: right; text-align: center">
+        {% comment %}TODO MESSAGGIO ESTENSIONE{% endcomment %}
+        L'estensione è stata automaticamente approvata essendo decorsi trenta giorni, ai sensi dell'articolo 9.5 del "Regolamento sull'organizzazione, le attività. la formazione e l'ordinamento dei volontari"
```

Comment:
9.5 riguarda i trasferimenti, non le estensioni. Il regolamento inoltre non prevede il termine dei 30 giorni per le estensioni, ma l'impossibilita' di negazione (meno che il motivo sia scemo, o mancante). In breve, si tratta di un nulla osta. Trasformiamo in:

> L'estensione è stata automaticamente approvata in mancanza di obiezione, ai sensi dell'articolo 10.3 del "Regolamento sull'organizzazione, le attività, la formazione e l'ordinamento dei volontari".